tests/test_version.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
+
+__author__ = "Shereen Elsayed (s_elsayed@ucsb.edu)"
+__description__ = "Test for version command"
+
+import subprocess
+from hop import version
+
+
+def test_version():
+    process = subprocess.run(
+        ['hop', 'version'], stdout=subprocess.PIPE, universal_newlines=True)
+
+    versions = process.stdout
+    packages = version.get_packages()
+    assert any(package in versions for package in packages)
```

Comment:
Shouldn't this be `all` instead of `any`?

Reply:
I didn't see your comment before I merge.
Well, yeah it seems that all is better. I will fix it and remerge.
Nice catch! Thanks